### PR TITLE
adding populate_dividends functionality

### PR DESCRIPTION
**Pull request by YugineS**

### CODE REVIEW

File: datawell/iex.py [hunk 1]
```diff
@@ -72,6 +73,17 @@ def get_cash_flow(self, symbol: str, params=None):
         cash_flow = self.load_from_iex(uri)
         return cash_flow
 
+    def populate_dividends(self, ticker: dict = None, range: str = '1y'):
+        if ticker:
+            ticker['dividends'] = self.load_devidends(ticker['symbol'], range)
+        else:
+            for ticker in self.stock_list:
+                self.populate_dividends(ticker, range)
```
Comment (nikolai-riabinin-epam) [inline datawell/iex.py]: I think ticker here will be a dictionary, not string
can you check it, please?

Comment (petelind) [reply datawell/iex.py]: @YugineS , ping :)

**Comment by astepanovv** [comment]
I added a PR #19  with improved error handling when retrieving data.
Could you please review it and implement similar in your code?

File: datawell/iex.py (hunk 1 above)
Comment (YugineS) [reply datawell/iex.py]: It is a dictionary. I do not see any discrepancies here.

**Pull request merged**